mobile/mobile.scss

```diff
@@ -17,3 +17,13 @@
     display: none;
   }
 }
+
+.dc-hero {
+  .btn {
+    width: 100%;
```

Comment:
should we use an utility class for this?

Reply:
As I am doing it only in mobile I choose to do it using CSS. Otherwise, I have to have a mobile layout for the hero, isn't?